pre-docs/writing_plans.md

```diff
@@ -351,6 +352,13 @@ $r = run_command('whoami', 'localhost,local://0.0.0.0')
 $r.to_data.each |$result_hash| { notice($result_hash['result']['stdout']) }
 ```
 
+You can also use `filter_set` to filter a ResultSet and apply a ResultSet function to the output:
```

Comment:
Maybe call out that `.targets` is a method available on a `ResultSet` (something you would not get with an array).